src/config.py

```diff
@@ -0,0 +1,32 @@
+import configparser
+import sys
+
+from src import printcolors as pc
+
+try:
+    config = configparser.ConfigParser(interpolation=None)
+    config.read("config/credentials.ini")
+except FileNotFoundError:
+    pc.printout("Error: file \"config/credentials.ini\" not found!", pc.RED)
+    pc.printout("\n")
```

Comment:
```suggestion
    pc.printout('Error: file "config/credentials.ini" not found!\n', pc.RED)
```
Readability: Use double quotes inside single quotes.

Reply:
I agree - I was trying to stay consistent with the style of the rest of the codebase

Reply:
Ok, we will keep the @cclauss version